mmv1/products/clouddeploy/DeployPolicy.yaml

```diff
@@ -175,71 +175,19 @@ properties:
                 - name: oneTimeWindows
                   type: Array
                   description: "Optional. One-time windows within which actions are restricted."
+                  custom_expand: "templates/terraform/custom_expand/clouddeploy_policy_onetime_windows.go.tmpl"
+                  custom_flatten: "templates/terraform/custom_flatten/clouddeploy_policy_onetime_windows.go.tmpl"
                   item_type:
                     type: NestedObject
                     properties:
-                      - name: "startDate"
-                        type: NestedObject
-                        description: "Required. Start date."
+                      - name: "start"
```

Comment:
what's the reasoning behind wanting to change it to just a string? This would be a breaking change due to removing the original nestedObject to instead just be string inputs. Is the current method not supporting a specific use case?

Reply:
The intention is to make it easier for user to set the date time.  To avoid the breaking change, if I keep the existing fields(defined in the API) and add the two new string fields, what is the best way to flatten the rules array to pass the acceptance test? I tried to write custom flatten in various ways but always get `ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.` error.